.github/workflows/download_asset.yaml

```diff
@@ -0,0 +1,117 @@
+name: download_asset
```

Comment:
The name is a bit misleading, maybe `building_assets`?

Reply:
Hm, yeah, it's the name of the example package, because we're in a mono repo here. I agree with you that if it's not a mono repo it should probably be named `building_assets`. Should we name it `package_download_asset.yaml` and `name: package_download_asset`?

Reply:
Sure, but it's just a nit. So do as you want :)